apps/glances/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schema.json",
+  "name": "Glances",
+  "port": 8420,
+  "available": true,
+  "exposable": true,
```

Comment:
```suggestion
  "exposable": false,
```
There should be no reason to expose this with a Public Domain IMO. Can confirm with @meienberger

Reply:
Sounds good, no disagreement.